README.md

```diff
@@ -1,2 +1,51 @@
 # conda-tracker
-Link git repositories and patches into an aggregate repository
+
+conda-tracker is a tool that links git repositories and patches into an aggregate 
+repository. By grouping repositories and patches into a single repository, tasks
+such as updating conda recipes prior to submitting to conda-concourse-ci become
+much easier.
+
+## Installation
+
+To install conda-tracker with conda:
+
+    $  conda install conda-tracker
+
+To install conda-tracker from source:
+
+    $  git clone https://github.com/conda/conda-tracker.git
+    $  cd conda-tracker
+    $  python setup.py install
+
+## Usage
+
+conda-tracker provides a CLI wrapped around a library. The CLI may be invoked
+as follows:
+
+    usage: conda-tracker [OPTIONS] COMMAND [ARGS]...
+    
+    commands:
+      add           Add a repository to the aggregate repository
+      patch         Apply a patch to the corresponding repository in the directory
+      update        Update a repository to a specific branch or to the master branch
+    
+    positional arguments:
```

Comment:
These seem like they should be grouped more specifically with the commands they go with?  Also, are any of them optional?  Could they be?  Seems like "master" is a sane default branch?